flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/ExecutionJobVertex.java

```diff
@@ -203,6 +204,27 @@ public JobVertexID getJobVertexId() {
 	public ExecutionVertex[] getTaskVertices() {
 		return taskVertices;
 	}
+
+	public Map<String, List<ExecutionVertex>> getTaskVerticesByTaskManager() {
```

Comment:
Can we move this to the handler? I think that will be the only place where we use it in the future as well.